srcpkgs/pear/template

```diff
@@ -0,0 +1,81 @@
+# Template file for 'pear'
+pkgname=pear
+version=1.10.12
+revision=1
+_version_xml_util=1.4.5
+_version_console_getopt=1.4.3
+_version_archive_tar=1.4.13
+_version_structures_graph=1.1.1
+_version_php_archive=0.14.0
+wrksrc="pear-core-${version}"
+hostmakedepends="php tar"
+depends="php"
+conf_files="/etc/php/pear.conf"
+short_desc="PEAR - PHP Extension and Application Repository"
+maintainer="eater <=@eater.me>"
+license="BSD-2-Clause"
+homepage="https://pear.php.net/"
+distfiles="
+	https://github.com/pear/pear-core/archive/refs/tags/v${version}.tar.gz
+	http://download.pear.php.net/package/PEAR-${version}.tgz
+	http://download.pear.php.net/package/XML_Util-${_version_xml_util}.tgz
+	http://download.pear.php.net/package/Console_Getopt-${_version_console_getopt}.tgz
+	http://download.pear.php.net/package/Archive_Tar-${_version_archive_tar}.tgz
+	http://download.pear.php.net/package/Structures_Graph-${_version_structures_graph}.tgz
+	http://download.pear.php.net/package/PHP_Archive-${_version_php_archive}.tgz"
+checksum="775e04b3897531852454638aa24c2a38627241c97884faf10ad9bf7ca9faf65a
+	df363c5e6091334c43f560eec2f220be2f8044eb5d422204a715adc4663ba923
+	e0f8736cb47ce9dd32814de45425ff03ad55a72ba8bb757e42c456f861feedf6
+	54bdfb7c2c958cbd7e1e8f1b964b95c3bfbf3b2779052523011b4ee49d7dfacd
+	2752e19121573a9e17be9514f855785df9d56768ebdeade85588913e502e829e
+	3b19abac379883f095c11fe2e5550cb61691b868f3d96573f6ed564be82fa08e
+	59fd3aead80bdd9fec6c232710754b92516a36c2845e0745b96c4aea9d2d5768"
+skip_extraction="PEAR-${version}.tgz
+	XML_Util-${_version_xml_util}.tgz
+	Console_Getopt-${_version_console_getopt}.tgz
+	Archive_Tar-${_version_archive_tar}.tgz
+	Structures_Graph-${_version_structures_graph}.tgz
+	PHP_Archive-${_version_php_archive}.tgz"
+replaces="php-pear>=0"
+
+post_extract() {
+	mkdir -p go-pear-tarballs/src;
+	for pear in ${XBPS_SRCDISTDIR}/${pkgname}-${version}/*.tgz; do
+		# PHP_Archive is only needed for creation
+		if grep -q 'PHP_Archive' <<< "$pear"; then
+			tar -C go-pear-tarballs/src -xf $pear
+			continue
```

Comment:
You should probably use `case` or `[[ ]]` bash conditionals here ;)

I think the bash conditional would look like `if [[ "$pear" = *"PHP_Archive"* ]]`

Reply:
Also, consider using `bsdtar` directly, if it's possible :)

Reply:
what do you mean by "use `bsdtar` directly"?

Reply:
You probably don't need to put tar in hostmakedeps, can just use bsdtar in these lines and it should work. And bsdtar is always available in the chroot.

Reply:
updated in https://github.com/void-linux/void-packages/pull/31679